it/uk/gov/hmrc/transitmovementsrouter/connectors/PushNotificationConnectorSpec.scala

```diff
@@ -94,7 +100,10 @@ class PushNotificationConnectorSpec
 
       stub(OK)
 
-      whenReady(connector.post(movementId, messageId, source).value) {
+      implicit val system: ActorSystem             = ActorSystem("test")
```

Comment:
Pass TestActorSystem as a trait instead of creating the materialiser here

Reply:
Okay